homeassistant/components/light/zwave.py

```diff
@@ -107,8 +107,8 @@ def update_properties(self):
 
     def _value_changed(self, value):
         """Called when a value has changed on the network."""
-        if self._value.value_id == value.value_id or \
-           self._value.node == value.node:
+        if self._value.value_id != value.value_id or \
+           self._value.node != value.node:
```

Comment:
That's not true.

It used to be that when they were NOT equal, we would return. Ignoring the update: [source](https://github.com/turbokongen/home-assistant/blob/6477122b23d70666fa66035aff942692ea979aad/homeassistant/components/light/zwave.py#L110-L111)

Now it is: if anything but this light has changed, do this. 

The difference is in the return statement. My preference is to filter out right away and return if not a related change.